cmake/projects/NASM/schemes/url_sha1_nasm_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
Is there a way to change the install prefix? I want to install the unpacked binaries into `bin`(which is the behavior of `make install`) instead of root of the prefix.